### ISSUE TITLE
When using SetRowHeightRange error when opening new XLSX file in Excel with Sheets.xml

### ISSUE THREAD

**Issue by rawilliamsjr** [ggreen86/XLSX-Workbook-Class#114]
I have attached the created XLSX files the first works fine but just adding the call to SetRowHeightRange triggers an error in Excel.
[Consolidated Pull Slip_1.xlsx](https://github.com/ggreen86/XLSX-Workbook-Class/files/14487341/Consolidated.Pull.Slip_1.xlsx)
[Consolidated Pull Slip_2.xlsx](https://github.com/ggreen86/XLSX-Workbook-Class/files/14487342/Consolidated.Pull.Slip_2.xlsx)

**Comment by rawilliamsjr** [comment]
[CPS.zip](https://github.com/ggreen86/XLSX-Workbook-Class/files/14562991/CPS.zip)

The zip contains the code , tables, and the resulting Excel workbook.

**Comment by ggreen86** [comment]
Rob--

Thank you for sending the zip file with the source to duplicate the problem.  I narrowed the failure to row 844 which is a blank line.  When writing the row information, it was not giving a correct range for cell selection on the row (it had 1:0 instead of 1:1).  The problem was not the range set for the row height, but rather this error due to no max column being set.  I have placed a MAX() command into the statement to ensure that 1 is the minimum value being written.

I have also fixed the methods SetPrintFitToHeight() and SetPrintFitToWidth() to now also set the fittopage setting correctly.  However, I have not incorporated your suggestion to have a separate SetPrintTitles() method.

Attached is the updated Release 41 beta 5.

Greg

[WorkbookXLSX R41 Beta 5.zip](https://github.com/ggreen86/XLSX-Workbook-Class/files/14564217/WorkbookXLSX.R41.Beta.5.zip)
